pkg/util/testing/wrappers.go

```diff
@@ -304,6 +310,24 @@ func (j *JobWrapper) Ready(ready int32) *JobWrapper {
 	return j
 }
 
+// DelTimestamp sets the job deletionTimestamp.
+func (j *JobWrapper) DelTimestamp(deletionTimestamp *metav1.Time) *JobWrapper {
```

Comment:
can we use the full name here please (`DeletionTimestamp`) for consistency across the wrappers?

Same with the resourceVersion below

Reply:
ya, that makes sense